src/widgets/edit/Conditional/ConditionsEdit/InputRow/index.js

```diff
@@ -0,0 +1,112 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import memoize from 'memoize-one';
+
+import DangerButton from '#rsca/Button/DangerButton';
+import ListView from '#rscv/List/ListView';
+import FaramGroup from '#rscg/FaramGroup';
+import SelectInput from '#rsci/SelectInput';
+
+import { iconNames } from '#constants';
+import _ts from '#ts';
+
+import Attribute from './Attributes';
+import styles from './styles.scss';
+
+const propTypes = {
+    index: PropTypes.number.isRequired,
+    item: PropTypes.object.isRequired, // eslint-disable-line react/forbid-prop-types
+    widgetData: PropTypes.object.isRequired, // eslint-disable-line react/forbid-prop-types
+    conditions: PropTypes.array.isRequired, // eslint-disable-line react/forbid-prop-types
+};
+
+const deleteClick = (rows, index) => {
+    const newRows = [...rows];
```

Comment:
Better use `filter()` instead of `splice()`. This way we need not re-create the array and again mutate the re-created array. Usage:
```
const deleteClick = (rows, index) => rows.filter((row, ind) => ind !== index);
```